modules/table/src/main/java/org/apache/ignite/internal/utils/RebalanceUtil.java

```diff
@@ -202,13 +222,46 @@ public static ByteArray stablePartAssignmentsKey(String partId) {
     }
 
     /**
-     * Extract table id from pending key of partition.
+     * Key that is needed for the rebalance algorithm.
+     *
+     * @param partId Unique identifier of a partition.
+     * @return Key for a partition.
+     * @see <a href="https://github.com/apache/ignite-3/blob/main/modules/table/tech-notes/rebalance.md">Rebalnce documentation</a>
+     */
+    public static ByteArray switchReduceKey(String partId) {
```

Comment:
Is this a partition id or an id of corresponding raft group?

Reply:
The latter one